sdk/src/main/java/io/dapr/exceptions/DaprException.java

```diff
@@ -151,6 +145,37 @@ public static void wrap(Exception exception) {
     return Mono.empty();
   }
 
+  /**
+   * Wraps an exception into DaprException (if not already DaprException).
+   *
+   * @param exception Exception to be wrapped.
+   * @return wrapped RuntimeException
+   */
+  public static RuntimeException propagate(Throwable exception) {
```

Comment:
We don't want to wrap throwable. It will catch stuff like https://docs.oracle.com/javase/7/docs/api/java/lang/OutOfMemoryError.html

Reply:
I've added ```Exceptions.throwIfFatal(exception);``` from project reactors.
Reactor deals with throwable wrapping with ```reactor.core.Exceptions#propagate```.

Reply:
OK.